### PR TITLE
git-annex: add configuration setting [annex].ENABLED

### PR DESCRIPTION
**Pull request by kousu**
Fixes https://github.com/neuropoly/gitea/issues/8

### CODE REVIEW

**Comment by kousu** [comment]
Corresponding patch to the related ansible deployment: https://github.com/neuropoly/computers/pull/420

**Comment by kousu** [comment]
This should be an easy review, @mguaypaq, if you have some time to put towards it.

I've put this underneath #6 though, so it's blocking that at the moment.

File: custom/conf/app.example.ini [hunk 1]
```diff
@@ -2360,6 +2360,15 @@ ROUTER = console
...
 ;PATH = data/lfs
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[annex]
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; Whether git-annex is enabled; defaults to false
+;ENABLED = false
```
Comment (mguaypaq) [inline custom/conf/app.example.ini:2370]: Thanks for this bit of documentation through example files!

File: modules/setting/annex.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package setting
+
+import (
+	"code.gitea.io/gitea/modules/log"
+)
+
+// LFS represents the configuration for Git LFS
```
Comment (mguaypaq) [inline modules/setting/annex.go:11]: Should update `LFS` -> `Annex`:
```suggestion
// Annex represents the configuration for git-annex
```

**Review by mguaypaq — APPROVED** [review]
LGTM, with one tiny suggestion below.

**Comment by kousu** [comment]
Thanks for the review!

**Pull request merged**